### PR TITLE
started playground page

### PR DESCRIPTION
**Pull request by GDayne**
in progress

### CODE REVIEW

File: General/Playground.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+> The title tells us that `11:55:11.561177 am` is an instance of the class `Time`. The top panel allows us to browse the instance variables of an object and their values. The bottom panel can be used to write expressions to send messages to the object. 
+
+5. __Basic inspect it__ `⇧⌘I` : Opens the classic inspector that offers a more minimal user interface and live updates of changes to the object.
+6. __Debug it__ `⇧⌘D` : Opens the debugger on the code.
+
+![Debug it on Time now](playground_debug_it.png)
+
+>The Debugger is a tool that not only lets you inspect the run-time stack of your program when an error is raised, but it also enables you to interact with all of the objects of your application, including the source code. In many cases you can modify your source code from the debugger and continue executing. The debugger is especially effective as a tool to support test-first development in tandem with SUnit.
+
+7. __Profile it__ : Profiles the code with the Pharo profile tool which shows how much time is spent for each message sent.
```
Comment (jecisc) [inline General/Playground.md]:
```suggestion
* __Profile it__ : Profiles the code with the Pharo profile tool which shows how much time is spent for each message sent.
```

File: General/Playground.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+8. __Code search__ offers several options provided by System Browser, such as browsing the source code of an expression, searching for senders and implementors, and so on.
```
Comment (jecisc) [inline General/Playground.md]:
```suggestion
* __Code search__ offers several options provided by System Browser, such as browsing the source code of an expression, searching for senders and implementors, and so on.
```

**Comment by jecisc** [comment]
@juliendelplanque I think we have enough to merge

**Comment by juliendelplanque** [comment]
Ok I'll merge.

**Pull request merged**